orchagent/aclorch.cpp

```diff
@@ -2839,6 +2906,10 @@ void AclOrch::doTask(Consumer &consumer)
         unique_lock<mutex> lock(m_countersMutex);
         doAclRuleTask(consumer);
     }
+    else if (table_name == CFG_ACL_TABLE_TYPE_TABLE_NAME || table_name == APP_ACL_TABLE_TYPE_TABLE_NAME)
+    {
+        doAclTableTypeTask(consumer);
```

Comment:
If we consume ```CFG_ACL_TABLE_TYPE_TABLE_NAME``` before ```CFG_ACL_TABLE_TABLE_NAME```, can we avoid the possible retry for custom ACL type?

Reply:
We have no control on the order in which we consume tables - we do not control how AclOrch::doTask() is called on which tables in which order. AclOrch::doTask() is called on a single table update and we don't know if it will be ACL_TABLE first or ACL_TABLE_TYPE first.